app/controllers/ProcessReviewController.scala

```diff
@@ -66,4 +66,19 @@ class ProcessReviewController @Inject() (reviewService: ReviewService, cc: Contr
     }
   }
 
+  def approval2iReviewPageComplete(id: String, pageUrl: String): Action[JsValue] = Action.async(parse.json) { request =>
+    def save(reviewInfo: ApprovalProcessPageReview): Future[Result] = {
```

Comment:
redundant {}

Reply:
still got my java head on ....... 20+ years of curly braces difficult to drop!!

Reply:
Done, in next commit